frontend/src/bundles/auth/store/slice.ts

```diff
@@ -5,18 +5,30 @@ import {
     type UserAuthResponseDto,
     type ValueOf,
 } from '~/bundles/common/types/types.js';
+import { type UserBonusGetAllItemResponseDto } from '~/bundles/users/users.js';
 
-import { logout, refreshUser, signIn, signUp, updateUser } from './actions.js';
+import {
+    loadAllUserBonusesTransactions,
+    logout,
+    refreshUser,
+    signIn,
+    signUp,
+    updateUser,
+} from './actions.js';
 
 type State = {
     dataStatus: ValueOf<typeof DataStatus>;
     user: UserAuthResponseDto | null;
+    userBonusesTransactions: UserBonusGetAllItemResponseDto[];
+    userBonusesStatus: ValueOf<typeof DataStatus>;
     isRefreshing: boolean;
 };
 
 const initialState: State = {
     dataStatus: DataStatus.IDLE,
+    userBonusesStatus: DataStatus.IDLE,
```

Comment:
I think if we have separate data status this is indication that worth to create separate slice for this data. But I'm not sure if this is priority for now, maybe we shouldn't focus on this.
@nikita-remeslov What do you think?

Reply:
Yes, looks like we should create a new slice in such cases. 
Do we need this additional status @ArtemLymarenko ?

Reply:
Yes, we can't use the datastatus